pkg/image-manager/entities.go

```diff
@@ -68,15 +45,13 @@ type SystemDependencies struct {
 
 // RuntimeDependencies defines a runtime dependency type
 type RuntimeDependencies struct {
-	Format   string `json:"format"`
 	Manifest string `json:"manifest"`
 }
 
 // Image defines an image type
 type Image struct {
 	entitystore.BaseEntity
 	DockerURL           string              `json:"dockerUrl"`
-	Language            Language            `json:"language"`
```

Comment:
It's not bad to keep this, as it's effectively caching the value so you don't need to look it up on the base image.

Reply:
Keeping language as informative field in the latest revision of this PR